pkg/datapath/linux/route/route.go

```diff
@@ -88,12 +88,11 @@ func (r *Route) ToIPCommand(dev string) []string {
 func lookupDefaultRoute(family int) (netlink.Route, error) {
 	routes, err := netlink.RouteListFiltered(family, &netlink.Route{Dst: nil}, netlink.RT_FILTER_DST)
 	if err != nil {
-		log.WithError(err).Error("Unable to list direct routes")
```

Comment:
Is this log message printed elsewhere or just not useful?

Reply:
After patch its rolled up to daemon and printed in Fatal message above. I've also added in update to the other caller so now both call traces here will print this in the Fatal message.